app/views/offers/confirm_accept.html.erb

```diff
@@ -5,13 +5,14 @@
 	<div class="panel-body">
 
 		<p>Are you sure you want to <strong>accept</strong> this offer?</p>
-
+		<% if @after_swap_offer_patrol.present? %>
 		<% if @after_swap_request_patrol[:result] && @after_swap_offer_patrol[:result]%>
```

Comment:
I'm not sure about this - we still want to check the awards before the swap to make sure the incoming offer has awards that would cover the outgoing person.

Can you change to allow this?